lib/api/globals/index.ts

```diff
@@ -6,30 +6,20 @@ import {
 } from "@/gql/graphql";
 import { fallbackLng } from "@/lib/i18n/settings";
 import queryAPI from "@/lib/api/client/query";
-import { linkFragment } from "@/lib/api/fragments/link";
-import {
-  footerFragment,
-  contactFormFragment,
-} from "@/lib/api/fragments/global";
 import { categoriesFragment } from "@/lib/api/fragments/categories";
 import { getSiteFromLocale } from "@/lib/helpers/site";
 import tags from "../client/tags";
 
 export async function getGlobalData(locale = fallbackLng) {
```

Comment:
this is a big untyped query, pulling out stuff where we can is a good start to getting it typed and understood